ofl/mplus1/METADATA.pb

```diff
@@ -0,0 +1,24 @@
+name: "Mplus 1"
+designer: "MPlus Fonts"
```

Comment:
This should match the attribution on the existing M PLUS fonts (eg https://fonts.google.com/specimen/M+PLUS+Rounded+1c)